Gemfile.lock

```diff
@@ -1,14 +1,20 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    rainbow (2.0.0)
-    sass (3.4.16)
-    scss_lint (0.40.1)
-      rainbow (~> 2.0)
-      sass (~> 3.4.1)
+    rake (11.1.2)
```

Comment:
It gets added when I run 'bundle'. Could be because of the upgrade I did on the Gemfile?